core/primitives/src/types.rs

```diff
@@ -1139,6 +1139,9 @@ pub enum ValidatorKickoutReason {
     DidNotGetASeat,
     /// Validator didn't produce enough chunk endorsements.
     NotEnoughChunkEndorsements { produced: NumBlocks, expected: NumBlocks },
+    /// Validator's last block proposal was for a protocol version older than
+    /// the network's voted protocol version.
+    ProtocolVersionTooOld { version: ProtocolVersion, network_version: ProtocolVersion },
```

Comment:
not sure why this is impacting the `check-protocol-schema`

Reply:
Discussion link: https://near.zulipchat.com/#narrow/channel/295558-core/topic/protocol.20version.20based.20validator.20kickout.20impl.20questions/near/514981213